py3status/modules/usbguard.py

```diff
@@ -1,195 +1,77 @@
     format_device_separator: show separator if more than one (default ' ')
-    format_notification_message: specify notification message to use
-        (default 'USB ID: {usb_id}\\nName: {name}\\nPort: {port}')
-    format_notification_title: specify notification title to use
-        *(default 'USB Device [\?if=policy=allow Allowed]'
-        '[\?if=policy=block Blocked][\?if=policy=reject Rejected]')*
-    permanent: specify behavior to use, otherwise auto False (default None)
 
 Format placeholders:
     {device}                  number of USB devices
-    {format_button_filter}    button to toggle USB device display filters
-    {format_button_permanent} button to toggle permanent states
     {format_device}           format for USB devices
 
-format_button_filter:
-    {filter}      USB device filter, eg None, any, allow, block
-
-format_button_permanent:
-    {permanent}   permanent boolean, eg False, True
-
 format_device:
-    {id}                   eg 1, 2, 5, 6, 7, 22, 23, 33
-    {policy}               eg allow, block
-    {usb_id}               eg 054c:0268
-    {name}                 eg Poker II, PLAYSTATION(R)3 Controller
-    {serial}               eg 0000:00:00.0
-    {port}                 eg usb1, usb2, usb3, 1-1, 4-1.2.1
-    {interface}            eg 00:00:00:00 00:00:00 00:00:00
-    {hash}                 eg ihYz60+8pxZBi/cm+Q/4Ibrsyyzq/iZ9xtMDAh53sng
-    {parent_hash}          eg npSDT1xuEIOSLNt2RT2EbFrE8XRZoV29t1n7kg6GxXg
-    {format_button_allow}  format for allow button
-    {format_button_block}  format for block button
-    {format_button_reject} format for reject button
-
-format_notification_*:
-    See `format_device`
+    {format_button_allow}     button to allow the device
+    {format_button_reject}    button to reject the device
+    {id}                      eg 1, 2, 5, 6, 7, 22, 23, 33
+    {policy}                  eg allow, block, reject
+    {usb_id}                  eg 054c:0268
+    {name}                    eg Poker II, PLAYSTATION(R)3 Controller
+    {serial}                  eg 0000:00:00.0
+    {port}                    eg usb1, usb2, usb3, 1-1, 4-1.2.1
+    {interface}               eg 00:00:00:00 00:00:00 00:00:00
+    {hash}                    eg ihYz60+8pxZBi/cm+Q/4Ibrsyyzq/iZ9xtMDAh53sng
+    {parent_hash}             eg npSDT1xuEIOSLNt2RT2EbFrE8XRZoV29t1n7kg6GxXg
 
 Requires:
     python-gobject: Python Bindings for GLib/GObject/GIO/GTK+
     usbguard: USB device authorization policy framework
 
-Examples:
-```
-# specify a list of filters: None, any, allow, block
-usbguard {
-    * None: show any devices with any options
-    * any: show any devices
-    * allow: show authorized devices
-    * block: show unauthorized devices
-}
-
-# show block device only, hide button
-usbguard {
-    format_button_filter = ''
-    filters = ['block']
-}
-```
-
-@author Cyril Levis (@cyrinux)
+@author @cyrinux, @lasers, @maximbaz
```

Comment:
Over 200 lines deleted. Plz take me off the list. Thx. Take the credit for simple usbguard. :-)

Reply:
Yes, please. I don't want to try and put my name everywhere. If it wasn't for the initial author, we wouldn't have this module. I just helped with getting things moving for initial author. I'm removing myself from your `xrandr_rotate` module too in `cleanup 2` PR.

Reply:
Your wish is my command